EventListener/ExceptionListener.php

```diff
@@ -69,20 +85,38 @@ public function onKernelException($event)
                     break;
                 }
             }
+
             $this->dispatcher->removeListener(KernelEvents::CONTROLLER_ARGUMENTS, $controllerArgsListener);
         };
-        $this->dispatcher->addListener(KernelEvents::CONTROLLER_ARGUMENTS, $controllerArgsListener, -100);
+
+        $this->dispatcher->addListener(KernelEvents::CONTROLLER_ARGUMENTS, $controllerArgsListener);
 
         $this->exceptionListener->onKernelException($event);
     }
 
+    public function removeCspHeader(ResponseEvent $event): void
+    {
+        if ($this->innerExceptionListener instanceof ErrorListener) {
+            $this->innerExceptionListener->removeCspHeader($event);
+        }
+    }
+
+    public function onControllerArguments(ControllerArgumentsEvent $event)
+    {
+        if ($this->innerExceptionListener instanceof ErrorListener) {
+            $this->innerExceptionListener->onControllerArguments($event);
+        }
+    }
+
     /**
      * {@inheritdoc}
      */
     public static function getSubscribedEvents()
     {
-        return array(
-            KernelEvents::EXCEPTION => array('onKernelException', -100),
-        );
+        if (class_exists(ErrorListener::class)) {
```

Comment:
related to my comment above, `master` branch supports 4.4+ only, this shouldn't be true always?

Reply:
which means I rather need to check for the LegacyExceptionListener